src/sidebar/services/oauth-auth.js

```diff
@@ -282,7 +286,10 @@ export default function auth(
       tokenInfoPromise,
       oauthClient(),
     ]);
-    await client.revokeToken(token.accessToken);
+
+    if (token) {
+      await client.revokeToken(token.accessToken);
+    }
```

Comment:
I didn't find a code path where `token` was definitely going to be `null`, but it also wasn't obvious that it _couldn't_ be null and so TS complained about that.